crates/storage/src/structured_storage.rs

```diff
@@ -264,80 +371,50 @@ where
 #[cfg(feature = "test-helpers")]
 pub mod test {
     use crate as fuel_core_storage;
-    use crate::kv_store::StorageColumn;
+    use crate::kv_store::{
+        KeyValueInspect,
+        StorageColumn,
+    };
     use fuel_core_storage::{
-        kv_store::{
-            BatchOperations,
-            KeyValueStore,
-            Value,
-        },
+        kv_store::Value,
         Result as StorageResult,
     };
-    use std::{
-        cell::RefCell,
-        collections::HashMap,
-    };
+    use std::collections::BTreeMap;
 
-    type Storage = RefCell<HashMap<(u32, Vec<u8>), Vec<u8>>>;
+    type Storage = BTreeMap<(u32, Vec<u8>), Value>;
```

Comment:
Just curious - why did you change this from a `HashMap` to a `BTreeMap`? Is order important? Or perhaps just performance?